packages/contracts-periphery/contracts/universal/op-nft/AttestationStation.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Semver } from "@eth-optimism/contracts-bedrock/contracts/universal/Semver.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+/**
+ * @title AttestationStation
+ * @dev   Contract for creating attestations.
+ * @notice This contract is not yet audited.
```

Comment:
should I have this notice here?

Reply:
Going to add more words to the @notice describing the contract like L1CrossDomainMessenger has